### ISSUE TITLE
When you were younger what made you interested in programming?

### ISSUE THREAD

**Issue by dmueller39** [rachsmithcodes/ama#36]

**Comment by rachsmithcodes** [comment]
hi Daniel,

I actually didn't develop a really passionate interest in programming until I was about 24. Growing up I was fortunate enough to have a computer from age 5 and was quite obsessed with using it (although not coding on it). My family also got dial-up internet in 1998 and I was really in to that too. I was best in my IT/Programming class in senior high school (I think we used Visual Basic? it's hard to remember now) so thought it made sense that I would study Software Engineering at University. However my SE class was pretty uninspiring and I hadn't made the connection that the sites I was using every day (myspace etc) was built using programming concepts, I just couldn't see how the things I was learning could translate to an appealing career. I dropped out and pursued Human Resources instead.
When I was 24 I realised I wasn't happy with where I was heading in life and I decided to take up coding again, by this point "web design" was just turning into "front end development" so I thought I would be a web designer. After spending some time with JavaScript I realised that programming was far more interesting to me than writing CSS, so I pursued that path :)

Thanks for asking!
